### ISSUE TITLE
[Solved]Deprecated Task?

### ISSUE THREAD

**Issue by toddca** [microsoft/azure-pipelines-extensions#506]
I noticed when adding this task to my pipeline that the name is "[Deprecated] Deploy SQL DACPAC" - is this really deprecated and if so what is the recommended replacement with like functionality aka replacement?

Also I did notice when importing my build and release definitions from one environment into another this task is the only one that does not import. It appears like it is not recognized, Is this because its deprecated?

**Comment by toddca** [comment]
I believe I may be looking at two different extensions - I noticed the GUID on the deprecated is 70a3a82d-4a3c-4a09-8d30-a793739dc94f while you guys here use 063c7649-4934-49e8-9eac-37121fe7e350.

So now my question is, how can I get hold of this extension to deploy into my new environment? It is available in my older environment as I believe I installed from the marketplace - this extension does not appear to be on the marketplace from what I can tell.
